packages/manager/apps/public-cloud/src/components/walkMe/templates/prevButton.html

```diff
@@ -0,0 +1,3 @@
+<button class="mr-1 oui-button oui-button_icon-alone {{ visibility }}" data-role="prev">
```

Comment:
is this supposed to be `ARIA` ?

because `role="prev"` doesn't exists: https://www.w3.org/TR/wai-aria-1.1/#prev

:)